unit_tests/test_config.py

```diff
@@ -679,6 +679,42 @@ def test_22_get_none():
         val = conf.get(None)
         assert val is None
 
+    def test_23_convert_ami_multi_region(self):
+        # test converting with all supported regions
+        os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
+        os.environ['SCT_AMI_ID_DB_SCYLLA'] = 'scylla-5.1.4-x86_64-2023-01-22T17-58-39Z'
```

Comment:
Let's test with the loader AMIs

The specific scylla version would be gone in a year or two...

Also let's mark those tests as integration tests